docs/index.rst

```diff
@@ -778,3 +774,9 @@ The Python bindings also provide some additional configuration properties:
     mylogger.addHandler(logging.StreamHandler())
     producer = confluent_kafka.Producer({'bootstrap.servers': 'mybroker.com'}, logger=mylogger)
 
+.. note::
+   In Python client, ``logger`` configuration property is used for log handler instead of ``log_cb``.
```

Comment:
maybe: 

In Python client, the ``logger`` configuration property is used for log handler, not ``log_cb``.

Reply:
Done